packages/tooltip/src/Tooltip.test.js

```diff
@@ -1,12 +1,34 @@
-import { takeSnapshotsOf } from "@hig/jest-preset/helpers";
-
+import { anchorPoints } from "@hig/flyout/src";
+import Button from "@hig/button";
+import React from "react";
+import renderer from "react-test-renderer";
 import Tooltip from "./Tooltip";
 
 describe("tooltip/Tooltip", () => {
-  takeSnapshotsOf(Tooltip, [
+  [
```

Comment:
Why was `takeSnapshotsOf` removed? This looks like copy & paste. The changes to this file are adding a lot of unnecessary boilerplate.

Reply:
Yeah, bad copy/paste.  Updated in 22208a4